README.md

```diff
@@ -84,9 +84,9 @@ Apache Airflow is tested with:
 
 |                      | Main version (dev)   | Stable version (2.2.3)   |
 | -------------------- | -------------------- | ------------------------ |
-| Python               | 3.6, 3.7, 3.8, 3.9   | 3.6, 3.7, 3.8, 3.9       |
-| Kubernetes           | 1.20, 1.21           | 1.18, 1.19, 1.20         |
-| PostgreSQL           | 10, 11, 12, 13       | 9.6, 10, 11, 12, 13      |
+| Python               | 3.7, 3.8, 3.9        | 3.7, 3.7, 3.8, 3.9       |
+| Kubernetes           | 1.20, 1.21           | 1.20, 1.21               |
+| PostgreSQL           | 10, 11, 12, 13       | 10, 11, 12, 13           |
```

Comment:
```suggestion
| Python               | 3.7, 3.8, 3.9        | 3.6, 3.7, 3.8, 3.9       |
| Kubernetes           | 1.20, 1.21           | 1.18, 1.19, 1.20         |
| PostgreSQL           | 10, 11, 12, 13       | 9.6, 10, 11, 12, 13      |
```

I think these should stay for 2.2.3, they won't get changed until 2.3.0 I believe. (Am I right @potiuk?)

Reply:
> I think these should stay for 2.2.3, they won't get changed until 2.3.0 I believe. (Am I right @potiuk?)

Technically speaking - Kubernets and Postgres are already changed for 2.2.3 according to our agreed rules. We released 2.2.3 after we dropped support for those. There is a slight doubt if we can drop support in minor version - but to be honest I see no problem with that - see my comment 
 https://github.com/apache/airflow/blob/main/dev/README_RELEASE_AIRFLOW.md#optional-prepare-new-release-branches-and-cache

Reply:
Same - Python 3.6 should be dropped in 2.2.4 IMHO if we release it.

Reply:
Explanation: We have not stated in our rules that it's the next "minor" release - it's the "next" release, so it could also be patchlevel.

Reply:
AH NO ! I WAS WRONG! We actually specified MINOR OR MAJOR.  So you are right @jedcunningham - those shoudl not be changed just yet.

Reply:
(Comoment to sefl @potiuk  - read before you speak from the top of your head)

Reply:
> We drop support for Python and Kubernetes versions when they reach EOL. We drop support for those EOL versions
>  in main right after EOL date, and it is effectively removed when we release the first new MINOR (Or MAJOR if there is
>  no new MINOR version) of Airflow For example, for Python 3.6 it means that we drop support in main right after
>  23.12.2021, and the first MAJOR or MINOR version of Airflow released after will not have it.